### PR TITLE
Use official ROCm distribution 1.8.2

### PR DESCRIPTION
**Pull request by ezyang**
Signed-off-by: Edward Z. Yang <ezyang@fb.com>

### CODE REVIEW

**Comment by ezyang** [comment]
(jobs cancelled)

**Comment by petrex** [comment]
@bddppq

File: docker/caffe2/jenkins/common/install_rocm.sh [hunk 1]
```diff
@@ -1,92 +1,33 @@
...
-                   rocm-profiler \
-                   cxlactivitylogger
-}
-
-install_centos() {
-    echo "Not implemented yet"
-    exit 1
-}
-
-install_hip_thrust() {
```
Comment (bddppq) [inline docker/caffe2/jenkins/common/install_rocm.sh]: @petrex I think we still need this part?

**Comment by bddppq** [comment]
Let's just remove the nightly stuffs, but keep the ubuntu and centos distinction? (we need to add support for centos pretty soon).

**Comment by dagamayank** [comment]
> This PR is currently stuck because the new stack causes an LLVM OOM for the PyTorch ROCm build.

@iotamudelta

**Comment by iotamudelta** [comment]
@ezyang let's reduce the number of jobs by half for now. This will give us 16 GB / job on the nodes and should be plenty.

**Comment by ezyang** [comment]
rocm1.7.1 is 1.8.2 after an (accidentally) force docker rebuild; abandoning this patch

**Pull request closed without merging**